scripts/dpla_ingest_update.py

```
@@ -0,0 +1,102 @@
+
+if existing_tables == new_tables:
+    print('No changes detected. WordPress update skipped.')
+else:
+    def replacer(tables):
+        it = iter(tables)
+        def _replace(m):
+            return next(it)
+        return _replace
+
+    new_content  = table_pattern.sub(replacer(new_tables), content)
+    update_resp  = requests.post(
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)